api-reference/graphql/messaging/reply-to-thread.mdx

```diff
@@ -2,10 +2,7 @@
 title: 'Reply to threads'
 ---
 
-You can reply to a thread using the `replyToThread` mutation. The call to this mutation will be successful if:
-
-- the last message in the thread is an email from the customer OR
-- the thread was just created with [`createThread`](/api-reference/graphql/threads/create)
+You can reply to a thread using the `replyToThread` mutation.
```

Comment:
```suggestion
You can reply to a thread using the `replyToThread` mutation. This will work regardless if the last email in the thread is from a user or a customer.
```